vendor/modules.txt

```diff
@@ -149,6 +149,13 @@ github.com/hashicorp/golang-lru/simplelru
 github.com/imdario/mergo
 # github.com/inconshreveable/mousetrap v1.0.0
 github.com/inconshreveable/mousetrap
+# github.com/insomniacslk/dhcp v0.0.0-20201112113307-4de412bc85d8
+github.com/insomniacslk/dhcp/dhcpv4
+github.com/insomniacslk/dhcp/dhcpv6
+github.com/insomniacslk/dhcp/dhcpv6/server6
```

Comment:
Can we vendor only these? Vendoring everything seems to bring in stuff we surely do not use (e.g. `github.com/u-root/u-root`)

Reply:
Removing `github.com/u-root/u-root` causes the following error- 

```ERROR: /root/go/src/kubevirt.io/kubevirt/vendor/github.com/insomniacslk/dhcp/iana/BUILD.bazel:3:11: no such package 'vendor/github.com/u-root/u-root/pkg/uio': BUILD file not found in any of the following directories. Add a BUILD file to a directory to mark it as a package.
 - /root/go/src/kubevirt.io/kubevirt/vendor/github.com/u-root/u-root/pkg/uio and referenced by '//vendor/github.com/insomniacslk/dhcp/iana:go_default_library'
ERROR: Analysis of target '//cmd/virt-handler:virt-handler-image' failed; build aborted: Analysis failed```.

Anyway, I don't think it is a good idea to filter out manually files that are brought by `make deps-update`.